source/lib/predicates/predicate.ts

```diff
@@ -11,6 +11,9 @@ export interface Validator<T> {
 	// tslint:disable-next-line completed-docs
 	message(value: T, label?: string, result?: any): string;
 
+	// tslint:disable-next-line completed-docs
```

Comment:
Can you add a doc comments here? Should also mention the automatic fallback (https://github.com/sindresorhus/ow/pull/130/files#diff-b570ff7fed1f4f37bd9c3139465bb941R19).